cypress/features/regression/card.feature

```diff
@@ -1,50 +1,47 @@
 Feature: Card default component
   I want to test Card default component properties
 
-  Background: Open Card default component page
-    Given I open "Card" component page
-
   @positive
   Scenario Outline: I set Card component spacing to <spacing>
-    When I select card spacing to "<spacing>"
+    When I open default "Card" component in noIFrame with "card" json from "commonComponents" using "<nameOfObject>" object name
     Then Card component has <padding> padding and <margin> margin
     Examples:
-      | spacing | padding | margin |
-      | small   | 24      | 16     |
-      | medium  | 32      | 24     |
-      | large   | 48      | 32     |
+      | spacing | padding | margin | nameOfObject      |
+      | small   | 24      | 16     | cardSpacingSmall  |
+      | medium  | 32      | 24     | cardSpacingMedium |
+      | large   | 48      | 32     | cardSpacingLarge  |
 
   @positive
   Scenario Outline: Set the width of Card component to <width>
-    When I set width to "<width>"
+    When I open default "Card" component in noIFrame with "card" json from "commonComponents" using "<nameOfObject>" object name
     Then Card component has set width to "<width>"
     Examples:
-      | width |
-      | 550px |
-      | 999px |
+      | width | nameOfObject |
+      | 550px | width550     |
+      | 999px | width999     |
 
   @positive
   Scenario Outline: Set the width of Card component to outOfScope <width>
-    When I set width to "<width>"
+    When I open default "Card" component in noIFrame with "card" json from "commonComponents" using "<nameOfObject>" object name
     Then Card component has not set width to "<width>"
```

Comment:
```suggestion
    Then Card component has not set width to <width>
```

Reply:
no, we can't do that, because the input accepts `string`